packages/react-native-renderer/src/ReactFabricHostConfig.js

```diff
@@ -151,9 +151,9 @@ class ReactFabricHostComponent {
   }
 
   measureLayout(
-    relativeToNativeNode: number | Object,
```

Comment:
Yay!

Reply:
I realize it's not part of this change, but should the `warningWithoutStack` inside of this function (and the one in `packages/react-native-renderer/src/ReactNativeFiberHostComponent.js`) be `__DEV__` gated?

Reply:
warningWithoutStack internally has a dev check and is a no-op in dev.